doc/api/errors.md

```diff
@@ -1890,6 +1905,7 @@ A module file could not be resolved while attempting a [`require()`][] or
 [`zlib`]: zlib.html
 [ES6 module]: esm.html
 [Node.js Error Codes]: #nodejs-error-codes
+[Legacy Node.js Error Codes]: #legacy-nodejs-error-codes
```

Comment:
1. Linter issue: missing heading for this link?
2. Doc tools use underscores, not hyphens in section ids.

Reply:
doh! missed adding the h2 header, fixing it now. Also, realized that I don't really need the link definition at the end of the file, so removed it.